package com.backend.warehouse.payload.response;

public class JwtResponse {
	private String accessToken;
	private String type = "Bearer";
	private Long id;
	private String username;
	private String profileName;
	private String email;
	private String role;

	public JwtResponse(String accessToken, Long id, String username, String profileName, String email, String role) {
		this.accessToken = accessToken;
		this.id = id;
		this.username = username;
		this.profileName = profileName;
		this.email = email;
		this.role = role;
	}

	public String getAccessToken()                        { return accessToken; }
	public void   setAccessToken(String accessToken)      { this.accessToken = accessToken; }

	public String getTokenType()                          { return type; }
	public void   setTokenType(String tokenType)          { this.type = tokenType; }

	public Long   getId()                                 { return id; }
	public void   setId(Long id)                          { this.id = id; }

	public String getUsername()                           { return username; }
	public void   setUsername(String username)            { this.username = username; }

	public String getEmail()                              { return email; }
	public void   setEmail(String email)                  { this.email = email; }

	public String getProfileName()                        { return profileName; }
	public void   setProfileName(String profileName)      { this.profileName = profileName; }

	public String getRole()                               { return role; }
	public void   setRole(String role)                    { this.role = role; }
}
